### PR TITLE
Register react 

### PR DESCRIPTION
**Pull request by Abdelrahman-Rabi**
user can register through this form and data will be saved in DB User table.

with error handling for some inputs from user side.

---------------------------------
Register user

---------------------------------
image

### CODE REVIEW

File: front-end/src/App.js [hunk 1]
```diff
@@ -1,29 +1,14 @@
...
-}
-
-/* 
-// functional component
 const App = () => {
   return (
     <div>
-      <h1>HELLO WORLD</h1>
+      <h1>HELLO WOdRLD</h1>
+
```
Comment (Besslan) [inline front-end/src/App.js]: remove the empty line

File: front-end/src/components/Register.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    email: "",
+    password: "",
+    password2: "",
+    phone: "",
+  });
+
+  const [errors, setErrors] = useState({});
+
+  const handleChange = (e) => {
+    console.log(e);
```
Comment (Besslan) [inline front-end/src/components/Register.js]: remove the console log

File: front-end/src/components/Register.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    setValues({
+      ...values,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  const handleSubmit = (e) => {
+    // console.log("Values", values);
+
```
Comment (Besslan) [inline front-end/src/components/Register.js]: remove the commented out line and the empty line

File: front-end/src/components/Register.js [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    e.preventDefault();
+    setErrors(validate(values));
+    // console.log("ERR", errors);
+    if (Object.keys(errors).length) {
+      console.log("need more data");
```
Comment (Besslan) [inline front-end/src/components/Register.js]: remove the commented out console log

File: front-end/src/App.js (hunk 1 above)
Comment (Abdelrahman-Rabi) [reply front-end/src/App.js]: Done

File: front-end/src/components/Register.js (hunk 2 above)
Comment (Abdelrahman-Rabi) [reply front-end/src/components/Register.js]: Done

File: front-end/src/components/Register.js (hunk 3 above)
Comment (Abdelrahman-Rabi) [reply front-end/src/components/Register.js]: Done

File: front-end/src/components/Register.js (hunk 4 above)
Comment (Abdelrahman-Rabi) [reply front-end/src/components/Register.js]: Done

**Pull request merged**